test/e2e/dra/test-driver/app/kubeletplugin.go

```diff
@@ -87,32 +84,37 @@ func (ex *ExamplePlugin) getJSONFilePath(claimUID string) string {
 	return filepath.Join(ex.cdiDir, fmt.Sprintf("%s-%s.json", ex.driverName, claimUID))
 }
 
-// FileOperations defines optional callbacks for handling CDI files
+// PluginConfig defines optional callbacks for handling CDI files
 // and some other configuration.
-type FileOperations struct {
+type PluginConfig struct {
```

Comment:
Renaming this was overdue. Should have done it already earlier when adding `NumResourceInstances`...